docker/build/Dockerfile-prod

```
@@ -1,25 +1,36 @@
 # Use the official Docker Hub Ubuntu 18.04 base image
 FROM ubuntu:18.04
+MAINTAINER Timesketch <timesketch-dev@googlegroups.com>
 
-# Setup install environment and Timesketch dependencies
-RUN apt-get update && apt-get -y install \
-    apt-transport-https \
-    apt-utils \
-    ca-certificates \
-    git \
-    libffi-dev \
-    lsb-release \
-    software-properties-common \
-    python3-dev \
-    python3-pip \
-    python3-psycopg2 \
- && rm -rf /var/lib/apt/lists/*
+# Create container with:
+# docker build --no-cache --build-arg PPA_TRACK="[dev|stable]"
+
+ARG PPA_TRACK=stable
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -y update
+RUN apt-get -y install apt-transport-https apt-utils
+RUN apt-get -y install libterm-readline-gnu-perl software-properties-common
+RUN add-apt-repository -y ppa:gift/$PPA_TRACK
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+# Install dependencies
+RUN apt-get -y install ca-certificates lsb-release locales python3-psycopg2
 
 # Install Plaso
-RUN add-apt-repository ppa:gift/stable
-RUN apt-get update && apt-get -y install \
-    plaso-tools \
- && rm -rf /var/lib/apt/lists/*
+RUN apt-get -y install plaso-tools
+
+# Install Timesketch
+RUN apt-get -y install python3-timesketch
```

```suggestion
RUN apt-get -y install timesketch-server
```

Should we install timesketch-server here, as discussed offline?